nemo/collections/asr/models/ctc_bpe_models.py

```diff
@@ -63,27 +65,18 @@ def __init__(self, cfg: DictConfig, trainer=None):
 
         # Set the new vocabulary
         with open_dict(cfg):
-            if "params" in cfg.decoder:
-                cfg.decoder.params.vocabulary = ListConfig(list(vocabulary.values()))
-            else:
-                cfg.decoder.vocabulary = ListConfig(list(vocabulary.values()))
+            cfg.decoder.vocabulary = ListConfig(list(vocabulary.values()))
```

Comment:
OMG... We really had those cases all over the models...? wow...

Reply:
Yeah, Vahid took a lot of pain to add these. Glad they can go away (hopefully)